daras_ai_v2/meta_content.py

```diff
@@ -150,6 +161,7 @@ def meta_description_for_page(
     else:
         description = metadata.meta_description
 
+    description = MarkdownIt(renderer_cls=RendererPlain).render(description)
```

Comment:
would be nice to have a helper func here that's just maybe `escape_markdown()` given its now being used in multiple places!

Reply:
okay. `daras_ai/text_format.py` seems like the right place to keep it.

escaping has a different meaning e.g. escaping HTML is replacing HTML chars like `<` & `>` with `&lt;` & `&gt;`. `parse_markdown` or `md_to_text` reads better.

Reply:
missed opportunity to call it `unmarkdown()` 😆

Reply:
not missed yet... 👀